src/curl.cpp

```diff
@@ -343,6 +343,16 @@ static const long S3FSCURL_RESPONSECODE_NOTSET      = -1;
 static const long S3FSCURL_RESPONSECODE_FATAL_ERROR = -2;
 static const int  S3FSCURL_PERFORM_RESULT_NOTSET    = 1;
 
+// [NOTE]
+// Default path to mime.types file.
+// For macOS 10.15 Cataline, Apache2.4 is pre-installed.
+//
+#if defined(__APPLE__)
+static const std::string MIMETYPE_DEFAULT_PATH = "/etc/apache2/mime.types";
```

Comment:
Same here.

Reply:
Here, stop the judgment in APPLE, and change it so that it is detected in order of /etc/apach2/mime.typs from /etc/mime.types.